config/default_config.json

```diff
@@ -3,8 +3,10 @@
     "0BSD",
     "Apache-1.0",
     "Apache-2.0",
+    "Artistic-2.0",
```

Comment:
Seems fine to me, but I would like to wait for `#open-source` to respond first (hoping to clear it with legal there).

Reply:
Artistic-2.0 is a "copyleft" license, so it can't go in `acceptedLicenses`. It's in the ["ask" list](https://github.com/Brightspace/policy/blob/master/open-source/open-source.md#must-ask-for-approval), so we'll instead add it for the specific lib we want to use it for.